src/standard/buffer.rs

```diff
@@ -454,17 +488,48 @@ impl<T: OclPrm> Buffer<T> {
             len: len,
             _data: PhantomData,
         };
-        
+
         // if data.is_none() { try!(buf.cmd().fill(&[Default::default()], None).enq()); }
         if data.is_none() { try!(buf.cmd().fill(Default::default(), None).enq()); }
         Ok(buf)
     }
 
+    /// [UNTESTED]
+    /// Create a buffer linked to a GL buffer object (created with openGL)
+    ///
+    /// ## Errors
+    ///
+    /// Don't forget to `.cmd().gl_acquire().enq()` before using it and
+    /// `.cmd().gl_release().enq()` after.
+    ///
+    /// See the [`BufferCmd` docs](/ocl/ocl/build/struct.BufferCmd.html)
+    /// for more info.
+    ///
+    // FIXME: need GLuint type in place of gl_object ! did I import all types from cl_h.rs ?
```

Comment:
I will do that, I am sorry for that, my text editor configuration does this itself...